stable/kong/values.yaml

```diff
@@ -61,7 +57,7 @@ resources: {}
 readinessProbe:
   httpGet:
     path: "/status"
-    port: admin-ssl
+    port: admin
     scheme: HTTPS
```

Comment:
@shashiranjan84 shouldn't the scheme depend on `values. useTLS` too?

Reply:
@cilindrox its the default value and user would change the scheme if they don't `useTLS`